src/lib.rs

```diff
@@ -690,15 +690,22 @@ mod tests {
 
     proptest! {
         #[test]
+        #[ignore = "broken (control char)"]
         fn crosscheck_olpc_cjson(v in arbitrary_json()) {
-            use olpc_cjson::CanonicalFormatter;
```

Comment:
Ouch! Thanks for finding that...I should have been really suspicious that it passed, because the key ordering should have been different. Fixed!

Reply:
Was trying to use the trait for 'encode!' and only this test started to fail, it still took me a bit to understand WTF was happening